kratos/includes/model_part.h

```diff
@@ -1574,6 +1633,49 @@ class KRATOS_API(KRATOS_CORE) ModelPart : public DataValueContainer, public Flag
     ///@name Private Operations
     ///@{
 
+    /**
+     * @brief This method trims a string and returns the first component (constant version)
+     * @param rStringName The given name to be trimmed
+     * @return The first index
+     */
+    IndexType FirstComponentName(const std::string& rStringName) const
```

Comment:
as i understand these methods would not be needed if we go for having string access only in modelpart. Is that correct?

Reply:
Yep